packages/email-templates/src/templates/keyAirdropped.ts

```diff
@@ -9,6 +9,10 @@ export default {
 
 <p>A new membership (#{{keyId}}) to the lock <strong>{{lockName}}</strong> was just airdropped for you!</p>
 
+{{#if customContent}}
+  {{{customContent}}}
+{{/if}}
```

Comment:
```suggestion
{{#if customContent}}
<div class="custom-content">
  {{{customContent}}}
<div>
{{/if}}
```

and let's add custom CSS for `custom-content`. Probably inside of a "gray" box with padding and rounded corners, maybe even with a different font?

Reply:
**Example 1**

<img width="400" alt="Screenshot 2023-02-10 at 11 41 52" src="https://user-images.githubusercontent.com/20865711/218072733-35b91681-82e1-4c78-a98b-e8f6b8ed3308.png">

**Example 2** 

<img width="400" alt="Screenshot 2023-02-10 at 11 42 03" src="https://user-images.githubusercontent.com/20865711/218072756-cc207f0f-20b1-44fb-b449-28b3101df837.png">

Reply:
Please add a bit more padding around the custom content, and also, I think it would be a good idea to add the lock's icon in the top left corner (you can just use an `<img>` with the lock icon URL?

Reply:
<img width="400" alt="Screenshot 2023-02-10 at 16 26 06" src="https://user-images.githubusercontent.com/20865711/218130959-f040e154-da9a-4adc-b53f-4d1235b9e8f9.png">

<img width="400" alt="Screenshot 2023-02-10 at 16 26 38" src="https://user-images.githubusercontent.com/20865711/218130970-b29a38ff-991d-41db-8873-5b33e055feda.png">

Reply:
<img width="400" alt="Screenshot 2023-02-10 at 16 34 05" src="https://user-images.githubusercontent.com/20865711/218131406-477c2976-8925-419f-b11a-72af805b368e.png">